locales/holidays-ru_RU.json

```diff
@@ -1,89 +1,69 @@
 {
-  "2018": {
-    "11": {
-      "22": {
-        "emoji": "🦃",
-        "message": "Thank you!"
-      }
-    },
-    "2": {
-      "16": {
-        "emoji": "🐕",
-        "message": "新年好 / 新年好"
+  "fixed": {
+    "1": {
+      "1": {
+        "emoji": "🎆",
+        "message": "С Новым Годом!"
       }
     },
-    "5": {
-      "12": {
-        "emoji": "👩",
-        "message": "Thanks, Mom!"
+    "1": {
+      "7": {
+        "emoji": "❄️",
+        "message": "С Рождеством!"
       }
     },
-    "6": {
-      "16": {
-        "emoji": "👨",
-        "message": "Thanks, Dad!"
-      }
-    }
-  },
-  "2019": {
-    "11": {
-      "28": {
-        "emoji": "🦃",
-        "message": "Thank you!"
+    "2": {
+      "14": {
+        "emoji": "💘",
+        "message": "С Днем святого Валентина!"
       }
     },
     "2": {
-      "5": {
-        "emoji": "🐷",
-        "message": "新年好 / 新年好"
+      "23": {
```

Comment:
This has to be in the same `"2"` block as the 2/14 holiday above, since due to the way JSON works, it uses the last copy of a given key in each object.

<details><summary>A fixed copy of the file:</summary>

```json
{
  "fixed": {
    "1": {
      "1": {
        "emoji": "🎆",
        "message": "С Новым Годом!"
      }
    },
    "1": {
      "7": {
        "emoji": "❄️",
        "message": "С Рождеством!"
      }
    },
    "2": {
      "14": {
        "emoji": "💘",
        "message": "С Днем святого Валентина!"
      },
      "23": {
        "emoji": "🛡",
        "message": "С Днем защитника Отечества!"
      }
    },
    "3": {
      "8": {
        "emoji": "🌼",
        "message": "С 8 марта!"
      }
    },
    "5": {
      "1": {
        "emoji": "🚩",
        "message": "С праздником Весны и Труда!"
      },
      "9": {
        "emoji": "🎇",
        "message": "С Днем Победы!"
      }
    },
    "6": {
      "1": {
        "emoji": "🧒",
        "message": "С Днем защиты детей!"
      },
      "12": {
        "emoji": "🇷🇺",
        "message": "С Днем России!"
      }
    },
    "9": {
      "1": {
        "emoji": "👩‍🏫",
        "message": "С Днем знаний!"
      }
    },
    "11": {
      "4": {
        "emoji": "🤝",
        "message": "С Днем народного единства!"
      }
    }
  }
}
```

</details>